src/javascript/app/pages/user/metatrader/metatrader.js

```diff
@@ -210,7 +211,13 @@ const MetaTrader = (() => {
 
     const getAllAccountsInfo = (response) => {
         MetaTraderUI.init(submit, sendTopupDemo);
-        show_new_account_popup = Client.canChangeCurrency(State.getResponse('statement'), (response.mt5_login_list || []), false);
+        show_new_account_popup = Client.canChangeCurrency(
+            State.getResponse('statement'),
+            (response.mt5_login_list || []),
+            State.getResponse('trading_platform_accounts'),
+            Client.get('loginid'),
```

Comment:
passing `loginid` to `canChangeCurrency()` everywhere it's used according to the change of `canChangeCurrency`'s interface as part of https://github.com/binary-com/binary-static/pull/6814/files